webpack.config.js

```diff
@@ -1,23 +1,22 @@
 const path = require('path');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
 
```

Comment:
- [ ]  Your app is generating errors during compilation, and hence not working as intended. Kindly ensure to fix this issue:

![image](https://user-images.githubusercontent.com/12745474/154248087-f129481c-6260-4ad1-8960-333bceb0dd5c.png)
![image](https://user-images.githubusercontent.com/12745474/154248234-a2d0ab59-0c6e-46c5-8591-7746c798f52c.png)